### ISSUE TITLE
dnsmasq: failed to create listening socket for 10.0.0.1: Address already in use

### ISSUE THREAD

**Issue by Prince-1501** [sensepost/mana#107]
# Deprecation Notice

At Defcon 26 we talked about how we're deprecating mana-toolkit in favour of [bettercap](http://bettercap.org) and [berate-ap](https://github.com/sensepost/berate_ap). The core of mana, [hostapd-mana](https://github.com/sensepost/hostapd-mana) is still under active development. This repo isn't going anywhere, and can still be used, but it hasn't been actively maintained for some time. Keep an eye on [w1f1.net](https://w1f1.net/) for more info.

# tl;dr
Logging issues here is unlikely to get your problem solved.

**Comment by Prince-1501** [comment]
When i am trying to create an fake access point then it creates an this error how i will remove this ? please help me

**Comment by bvdx199** [comment]
i have the same problem. i upgraded my kali nethunter into a full kali linux but thn i deleted mana toolkit  by accicent so i took this repo. now i have 2 kindsof start-full-mana, one in usr/share/hostapd and one in sdcard/nh_files/hostapd i checked netstat -l and i see an tcp server listening on 10.0.0.1 but also a upnp on localhost:58904.

i have to close those but i dontn know how. did u solved yet?
